tidy3d/components/device/simulation.py

```diff
@@ -379,11 +479,34 @@ def plot_heat_conductivity(
             bounds=self.simulation_bounds, x=x, y=y, z=z, hlim=hlim, vlim=vlim
         )
 
-        cbar_cond = colorbar == "conductivity"
+        cbar_cond = True
 
-        ax = self.scene.plot_heat_conductivity(
-            ax=ax, x=x, y=y, z=z, cbar=cbar_cond, alpha=alpha, hlim=hlim, vlim=vlim
-        )
+        simulation_types = self._get_simulation_types()
+        if property is None:
```

Comment:
would it make sense to define this mapping from `DeviceSimulation` to plotting property as a `@property`?

Reply:
you mean something like 
```
property = self._plot_property
```
where `_plot_property` is `None` when `len(simulation_types)>1`?

Reply:
I actually misread the logic in this function. Probably  it's fine how it is.

I think in general I just think about ways to avoid code like
```
                if DeviceSimulationType.CONDUCTION in simulation_types:
                    property = "electric_conductivity"
                elif DeviceSimulationType.HEAT in simulation_types:
                    property = "heat_conductivity"
```

but it might be unavoidable here.

Reply:
yeah, in this case it's tricky to get rid of these since `DeviceSimulation` doesn't have a `type`. This is by design since we might define different types of simulation within this class. We could add a `simulation_type` to `DeviceSimulation` and then define the types of simulation possibilities so the user can choose what to run though this might be more restrictive. I think this is a design decision that maybe should be discussed somewhere else? what do you think @tylerflex @dbochkov-flexcompute ?